cli/src/types.ts

```diff
@@ -25,7 +25,8 @@ export const availablePackages = [
   'tamagui',
   'restyle',
   'unistyles',
-  'i18next'
+  'i18next',
+  'vexo-analytics',
```

Comment:
@sammoore As this adds a new option to the CLI, should we add something to that file you mentioned such that this doesn't break https://rn.new?